src/win/util.c

```diff
@@ -59,7 +59,7 @@ static char *process_title;
 static CRITICAL_SECTION process_title_lock;
 
 /* Interval (in seconds) of the high-resolution clock. */
-static double hrtime_interval_ = 0;
+static long hrtime_interval_ = 0;
```

Comment:
The return from `.QuadPart` is `LONGLONG` apparently. Same thing as `uint64_t`?

https://msdn.microsoft.com/en-us/library/windows/desktop/aa383713(v=vs.85).aspx